test/exprtest/blas1_iface_test.cpp

```diff
@@ -25,11 +25,11 @@
 
 #include "blas_test.hpp"
 
-typedef ::testing::Types<blas_test_args<double>> BlasTypes;
+typedef ::testing::Types<blas_test_float<>, blas_test_double<>> BlasTypes;
 
 TYPED_TEST_CASE(BLAS_Test, BlasTypes);
 
-REGISTER_SIZE(::RANDOM_SIZE, interface1_test)
+REGISTER_SIZE(1023, interface1_test)
```

Comment:
Could you make a comment explaining why this particular constant is used?

Reply:
Added